<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitorDialog</class>
 <widget class="QDialog" name="monitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>monitorDialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicPane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="graphicsSourceSelect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="grapicSourceLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Graphics source:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graphicRefreshButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="refreshingRateLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Refreshing rate (ms):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshRateSetButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="refreshRateEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="logPane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="pictureSelect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="connectionSettingsButton">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectionSettingsLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection settings:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
